demos/checkboxradio-checkbox/index.php

```diff
@@ -117,10 +117,13 @@
 		<h2>Enhanced</h2>
 
 		<div data-demo-html="true">
-			<div class="ui-checkbox">
-				<label for="checkbox-enhanced" class="ui-button ui-corner-all ui-button-inherit ui-checkbox-off">I agree</label>
-				<input type="checkbox" name="checkbox-enhanced" id="checkbox-enhanced" data-enhanced="true">
-			</div>
+			<form>
+				<label class="ui-checkboxradio-label ui-corner-all ui-button ui-widget ui-button-inherit">
+					<span class="ui-checkboxradio-icon ui-corner-all ui-icon ui-icon-background ui-icon-blank"></span>
+					<span class="ui-checkboxradio-icon-space"> </span>
+					<input type="checkbox" data-enhanced="true" name="checkbox-enhanced" class="ui-checkboxradio ui-helper-hidden-accessible">I agree
+				</label>
+			</form>			
```

Comment:
When you check the checkbox the icon gets the active color, but the check icon itself isn't added. Not sure if this is a problem with the markup or a bug in the widget.